src/pocketmine/Player.php

```diff
@@ -2266,7 +2266,7 @@ public function handleDataPacket(DataPacket $packet){
 					break;
 				}
 
-				if(!in_array($packet->protocol, ProtocolInfo::ACCEPTED_PROTOCOLS)){
+				if($packet->protocol !== ProtocolInfo::CURRENT_PROTOCOL){
```

Comment:
... to the best of your/our knowledge. Until we can say for 100% certain what protocol changes were made and what behaviours will result, we should not mix them. MCPE doesn't allow mixing them, and there's a damn good reason for that.